### PR TITLE
[CLU@BF] Samples for using CLU in a flight-booking bot

### PR DESCRIPTION
**Pull request by LouaiZahran**
## Purpose

Sample for using CLU with the Bot Framework for JavaScript

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: Sample
```

### CODE REVIEW

File: core-bot-with-clujs/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+```text
+CluProjectName="Your CLU project name"
+CluDeploymentName="Your CLU deployment name"
+CluAPIKey="Your CLU Subscription key here"
+CluAPIHostName="Your CLU App region here (i.e: westus.api.cognitive.microsoft.com)"
+```
+
+# To run the bot
+
```
Comment (AhmedLeithy) [inline core-bot-with-clujs/README.md:42]: Can we tell the user to navigate to the correct location first?

**Comment by AhmedLeithy** [comment]
Let's add this sample to the outer readme in the repo as well https://github.com/Azure-Samples/cognitive-service-language-samples

File: core-bot-with-clujs/deploymentTemplates/deployUseExistResourceGroup/readme.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Usage
```
Comment (AhmedLeithy) [inline core-bot-with-clujs/deploymentTemplates/deployUseExistResourceGroup/readme.md:1]: Did we write these deployment steps/templates? Or were they present in the original JS sample?

File: core-bot-with-clujs/dialogs/flightBookingRecognizer.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (AhmedLeithy) [inline core-bot-with-clujs/dialogs/flightBookingRecognizer.js]: This is great, but should I feel like it is misplaced in the "dialogs" folder. Should we have it in it's own folder instead for clarity?

**Review by AhmedLeithy — COMMENTED** [review]
Great work! Few comments here and there

File: core-bot-with-clujs/deploymentTemplates/deployUseExistResourceGroup/readme.md (hunk 2 above)
Comment (LouaiZahran) [reply core-bot-with-clujs/deploymentTemplates/deployUseExistResourceGroup/readme.md:1]: They were already present.

**Review by AhmedLeithy — APPROVED** [review]

**Pull request merged**